src/main/java/com/wdtinc/mapbox_vector_tile/build/MvtLayerProps.java

```diff
@@ -34,7 +34,15 @@ public Integer valueIndex(Object v) {
     public int addKey(String key) {
         Objects.requireNonNull(key);
         int nextIndex = keys.size();
-        final Integer mapIndex = keys.putIfAbsent(key, nextIndex);
+        // Android API 24 call
+        // final Integer mapIndex = keys.putIfAbsent(key, nextIndex);
```

Comment:
Let's make a 'putIfAbsent' utility function with the comment that it's to support android legacy.